backend/migrations/20210214110931_teachersProficiency.js

```diff
@@ -0,0 +1,21 @@
+exports.up = function(knex, Promise) {
+    return knex.schema.raw(`
+      ALTER TABLE "teachers" DROP CONSTRAINT "teachers_LanguageProficiency_check";
+      ALTER TABLE "teachers" ADD CONSTRAINT "teachers_LanguageProficiency_check" 
+      CHECK ("LanguageProficiency" IN ('Little', 'Simple', 'Intermediate', 'Fluent'));
+      ALTER TABLE "teachers" DROP CONSTRAINT "teachers_EnglishProficiency_check";
+      ALTER TABLE "teachers" ADD CONSTRAINT "teachers_EnglishProficiency_check" 
+      CHECK ("EnglishProficiency" IN ('Little', 'Simple', 'Intermediate', 'Fluent'))
+    `);
+  };
+
+  exports.down = async function(knex) {
+    return knex.raw(`
+        ALTER TABLE "teachers" DROP CONSTRAINT "teachers_LanguageProficiency_check";
+        ALTER TABLE "teachers" ADD CONSTRAINT "teachers_LanguageProficiency_check"
+        CHECK ("LanguageProficiency" IN ('Little', 'Simple', 'Intermediate', 'Fluent'));
+        ALTER TABLE "teachers" DROP CONSTRAINT "teachers_EnglishProficiency_check";
+        ALTER TABLE "teachers" ADD CONSTRAINT "teachers_EnglishProficiency_check"
+        CHECK ("EnglishProficiency" IN ('Little', 'Simple', 'Intermediate', 'Fluent'));
```

Comment:
This should be the opposite of what is defined in the "up" function. So I think here you should be removing the constraint?

Reply:
the `ADD CONSTRAINT` operations have now been removed.